components/TodoList.js

```diff
@@ -58,35 +66,97 @@ function TodoList() {
         value={newTodo}
         onChange={(e) => setNewTodo(e.target.value)}
       />
+      <select value={priority} onChange={handlePriorityChange}>
+        <option value="high">High Priority</option>
+        <option value="medium">Medium Priority</option>
+        <option value="low">Low Priority</option>
+      </select>
       <button onClick={addNewTodo}>Add</button>
-      <ul>
-        {todos.map((todo) => {
-          if (editTodoId === todo.id) {
-            return (
+
+      <div style={{ display: 'flex' }}>
+        <div style={{ flex: 1 }}>
+          <h3>High Priority</h3>
+          <ul>
+            {priorityGroups.high.map((todo) => (
+              <li key={todo.id}>
+                {editTodoId === todo.id ? (
+                  <>
+                    <input
+                      type="text"
+                      value={editedTodoText}
+                      onChange={(e) => setEditedTodoText(e.target.value)}
+                    />
+                    <button onClick={() => saveEditedTodo(todo.id)}>Save</button>
+                    <button onClick={cancelEditing}>Cancel</button>
+                  </>
+                ) : (
+                  <>
+                    {todo.text}
+                    <button onClick={() => removeTodo(todo.id)}>Delete</button>
+                    <button onClick={() => startEditing(todo.id, todo.text)}>Edit</button>
+                  </>
+                )}
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div style={{ flex: 1 }}>
```

Comment:
the same as the comment for line 76/77

Reply:
Thank you, it's a very good idea, I've now created a new folder for the styled components